pkg/kritis/crd/securitypolicy/violation.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package securitypolicy
+
+import (
+	"fmt"
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/constants"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+)
+
+type violation int
```

Comment:
Hmm, maybe this would be a bit nicer as `type violation string`, then your functions below can cast the strings into the `type violation`, that way we'd get a bit more type safety.